src/boundary_operators.jl

```diff
@@ -22,6 +22,35 @@ function rem1(idx,L)
 end
 
 
+#= LEAKY BCs =#
```

Comment:
Purely style remark: Leaky should come after the other ones, down here:

https://github.com/anriseth/PDEOperators.jl/blob/1879f32ced709fd4970d1a72dee9e1710e5553ce/src/boundary_operators.jl#L94